src/main/kotlin/at.phatbl.shellexec/logging/GradleLogOutputStream.kt

```diff
@@ -4,11 +4,12 @@ import org.apache.commons.exec.LogOutputStream
 import org.gradle.api.GradleException
 import org.gradle.api.logging.LogLevel
 import org.gradle.api.logging.Logger
+import java.io.File
 
 /**
  * Adapter which passes log output through to Gradle's logger.
  */
-class GradleLogOutputStream(val logger: Logger, val level: LogLevel): LogOutputStream(level.ordinal) {
+class GradleLogOutputStream(val logger: Logger, val level: LogLevel,val logFiles: Array<File>?): LogOutputStream(level.ordinal) {
```

Comment:
Add a space after the comma. Also, add a default argument of `null` to `logFiles`, so you don't have to explicitly pass null.
```suggestion
class GradleLogOutputStream(val logger: Logger, val level: LogLevel, val logFiles: Array<File>? = null): LogOutputStream(level.ordinal) {
```